server/main.lua

```diff
@@ -23,7 +23,11 @@ function RemoveOwnedVehicle(plate)
 end
 
 AddEventHandler('onResourceStart', function(resourceName)
-	Wait(1000)
+	Wait(420)
+	SQLVehiclesAndCategories()
```

Comment:
This will cb the function only when the database is ready and if the resource started was the current one. Otherwise, this will execute every time a resource has started.

```suggestion
AddEventHandler('onResourceStart', function(resourceName)
  if resourceName == GetCurrentResourceName() then
    MySQL.ready(function() SQLVehiclesAndCategories() end)
  end
end)
```